server/models/postgis/project.py

```diff
@@ -994,6 +994,8 @@ def as_dto_for_mapping(self, locale: str, abbrev: bool) -> Optional[ProjectDTO]:
             project_dto.organisation_name = project.organisation.name
             project_dto.organisation_logo = project.organisation.logo
 
+        project_dto.project_info_locales = ProjectInfo.get_dto_for_all_locales(self.id)
```

Comment:
@JorgeMartinezG thanks, it worked well. I think we can erase the lines 989 and 990, right?